app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a>
```

Comment:
Toàn bộ fle html.erb em check lại 2 vấn đề gồm `end-line` và `I18n` nhé bé bì

Reply:
Ngoài ra, em thay vì dùng tag `a` của html, em dùng `link_to` của form rails helper support hì